ui/apps/portal/src/plugins/Starboard/Starboard.tsx

```diff
@@ -35,6 +35,7 @@ await micropip.install('ssl')
 await micropip.install('pyjwt==2.9.0')
 await micropip.install('${uiFilesUrl}starboard-notebook-base/pyodidepyqe-0.0.2-py3-none-any.whl', keep_going=True)
 os.environ['PYQE_URL'] = '${MRI_ROOT_URL}/'
+os.environ['TOKEN'] = '${jwtToken}'
```

Comment:
re add token that was removed [here](https://github.com/OHDSI/d2e/pull/82/files#diff-75783a1e10543cead8c1b25570ff15aaa6830ac6b257f3c3018bdd39b1042fd5L98)